tests/organigram/test_helpers.py

```diff
@@ -32,10 +32,6 @@ def test_get_unit_merged_id_by_synonym(monkeypatch: MonkeyPatch) -> None:
     # Verify we get back a merged ID
     assert isinstance(merged_id, MergedOrganizationalUnitIdentifier)
     assert merged_id == mock_unit_id
-
-
-@pytest.mark.usefixtures("mocked_wikidata")
-def test_get_unit_merged_id_by_synonym_not_found(monkeypatch: MonkeyPatch) -> None:
     # Mock the load function
```

Comment:
```suggestion

    # Set-up another Mock to test for caching and unknown synonym
    # Mock the load function
```

suggestion: visually separate with an empty line and an explaining comment